mapie/tests/test_classification.py

```diff
@@ -267,30 +278,31 @@
 }
 
 COVERAGES = {
-    "score": 6 / 9,
-    "score_cv_mean": 1,
-    "score_cv_crossval": 1,
-    "cumulated_score_include": 1,
-    "cumulated_score_not_include": 5 / 9,
-    "cumulated_score_randomized": 6 / 9,
-    "cumulated_score_include_cv_mean": 1,
-    "cumulated_score_not_include_cv_mean": 5 / 9,
-    "cumulated_score_randomized_cv_mean": 5 / 9,
-    "cumulated_score_include_cv_crossval": 2 / 9,
-    "cumulated_score_not_include_cv_crossval": 0,
-    "cumulated_score_randomized_cv_crossval": 4 / 9,
-    "naive": 5 / 9,
-    "top_k": 1,
-    "raps": 1,
-    "raps_randomized": 8/9
+    'score': 0.6666666666666666,
+    'score_split': 0.8888888888888888,
+    'score_cv_mean': 1.0,
+    'score_cv_crossval': 1.0,
+    'cumulated_score_include': 1.0,
+    'cumulated_score_not_include': 0.5555555555555556,
```

Comment:
You should also test for the others method, to be as exhaustive as for the others functionnalities

Reply:
I agree, but this is outside the current RP. I only propose the `split`cv method. But before this PR, `top_k`, `raps` and `raps_randomized` methods already exist without exhaustive tests (only on `prefit`). So that this PR is not too long, I propose to create a question to solve this problem.

Reply:
Done.

Reply:
Okay, now it is done.